### PR TITLE
Separation branch

### PR DESCRIPTION
**Pull request by OlyaMilch**
Moving all classes into separate files.
Create map () in the Pecipe.py

### CODE REVIEW

File: ingredients.py [hunk 1]
```diff
@@ -71,18 +71,5 @@ def delete_ingredient(id_i: int):
...
-    def to_json(self):  # просто селф, т.к. мы УЖЕ внутри класса NetworkIngredient
-        data = json.dumps(self.__dict__)  # дикт возвращает все по ключам
-        return data
-
-
-class IngredientToNetworkIngredientMapper:
-    @classmethod  # не cоздаем объект от класса, а вызываем объект от класса
-    def map(cls, ingredient: Ingredient):  # cls-обозначает класс, а не объект
-        return NetworkIngredient(ingredient.id_i, ingredient.title, ingredient.category)  # возвращает созданный объект
+from class_NetworkIngredient import *
```
Comment (Makentoshe) [inline ingredients.py:74]: move imports to the top

File: Recipe.py [hunk 2]
```diff
@@ -68,3 +71,7 @@ def delete_recipe(id_r: int):
         db.session.commit()
     except Exception:
         return "Recipe not found"
+
+
+from class_NetworkRecipe import *
```
Comment (Makentoshe) [inline Recipe.py:76]: Move imports to the top

**Review by Makentoshe — CHANGES_REQUESTED** [review]
Remove class_ prefix for .py files.

**Comment by Pelmen323** [comment]
Why do you use the dot symbol in commit names? As for me, it looks odd

File: class_IngredientToNetworkIngredientMapper.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from class_NetworkIngredient import *
+from ingredients import *
+class IngredientToNetworkIngredientMapper:
+    @classmethod  # не cоздаем объект от класса, а вызываем объект от класса
+    def map(cls, ingredient: Ingredient):  # cls-обозначает класс, а не объект
+        return NetworkIngredient(ingredient.id_i, ingredient.title, ingredient.category)  # возвращает созданный объект
```
Comment (Pelmen323) [inline class_IngredientToNetworkIngredientMapper.py:6]: Add an empty line to the end of the file, you can see here that github also hints that

File: class_IngredientToNetworkIngredientMapper.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+from class_NetworkIngredient import *
```
Comment (Pelmen323) [inline class_IngredientToNetworkIngredientMapper.py:1]: Why do we use star imports?

File: class_IngredientToNetworkIngredientMapper.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+from ingredients import *
+class IngredientToNetworkIngredientMapper:
```
Comment (Pelmen323) [inline class_IngredientToNetworkIngredientMapper.py:3]: Add a space between class and import

File: ingredients.py [hunk 6]
```diff
@@ -43,8 +42,9 @@ def get_ingredient_by_id(id_i):
     ingredient = Ingredient.query.get(id_i)
     if ingredient is None:  # сначала проверка, потом маппер
         return 'Ingredient not found'
-    network_ingredient = IngredientToNetworkIngredientMapper.map(ingredient)
-    return network_ingredient.to_json(), 200
+    else:
```
Comment (Pelmen323) [inline ingredients.py:45]: why do we need 'else' here?

File: ingredients.py (hunk 1 above)
Comment (OlyaMilch) [reply ingredients.py]: fixed

File: Recipe.py (hunk 2 above)
Comment (OlyaMilch) [reply Recipe.py]: fixed

File: class_IngredientToNetworkIngredientMapper.py (hunk 4 above)
Comment (OlyaMilch) [reply class_IngredientToNetworkIngredientMapper.py]: fixed

File: class_IngredientToNetworkIngredientMapper.py (hunk 5 above)
Comment (OlyaMilch) [reply class_IngredientToNetworkIngredientMapper.py]: fixed

File: class_NetworkIngredient.py [hunk 7]
```diff
@@ -0,0 +1,10 @@
+import json
```
[the remark this answers is not in the document]
Comment (OlyaMilch) [reply class_NetworkIngredient.py]: fixed

File: ingredients.py (hunk 6 above)
Comment (OlyaMilch) [reply ingredients.py]: delete

File: ingredientmapper.py
[the remark this answers is not in the document]
Comment (OlyaMilch) [reply ingredientmapper.py:1]: all renamed

**Pull request merged**